README.md

```diff
@@ -462,7 +462,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
 
 ## Code Quality
 
-  * [SoftaCheck](https://www.softacheck.com/) — An online tool that performs static analysis for C/C++ code using open source tools such as cppcheck and clang-tidy and automatically generates code documentation for users using doxygen. This tool is free for use.
+  * [SoftaCheck](https://github.com/apps/softacheck) — An online tool that performs static analysis for C/C++ code using open source tools such as cppcheck and clang-tidy and automatically generates code documentation for users using doxygen. This tool is free for use.
```

Comment:
think we can just remove it at this point tbh.

Reply:
But the tool seems to be working.

Reply:
if they dont love it enough to fix their stuff, neither do i :)

Reply:
okay, I will the edit the pr with removal

Reply:
I also thought I should remove the tool before making the pr but then thought maybe you will say that the tool is working on the github , so, we should not remove.